timeline-chart/src/time-graph-container.ts

```diff
@@ -75,9 +75,13 @@ export class TimeGraphContainer {
     }
 
     // if canvas size has changed displayWidth need to be updated for zoomfactor
-    reInitCanvasSize(newWidth: number){
-        this.application.renderer.resize(newWidth, this.config.height);
+    reInitCanvasSize(newWidth: number, newHeight?: number) {
```

Comment:
You can initialize from the method's signature like this: newHeight: number = this.config.height